tests/glibc/tests.failed

```diff
@@ -2,3 +2,4 @@
 /glibc/build/inet/test-ifaddrs: creates a socket with AF_NETLINK
 /glibc/build/io/tst-ftw-lnk.d/link2-tgt ldso: /glibc/build/io/tst-ftw-lnk.d/link2-tgt: Not a valid dynamic program 
 /glibc/build/nptl/test-mutexattr-printers Makefile:57: recipe for target 'one' failed ****Runs without error in GDB****
+/glibc/build/nptl/tst-cancel8: Requires support of deferred cancel type for pthread_cancel
```

Comment:
Hey, since this is a test that flakily passes it would be better to add it to tests.remove since running the automated script to update this suite might cause one to unkowingly put it in our passed list again..

Reply:
Thanks for pointing that out! Didn't realize the difference between failed and remove lists till now.